src/commonMain/kotlin/exchange.dydx.abacus/validator/TriggerOrdersInputValidator.kt

```diff
@@ -175,18 +184,23 @@ internal class TriggerOrdersInputValidator(
             equityTier?.nextLevelRequiredTotalNetCollateralUSD
         val numOrders = orderCount(subaccount)
         var numOrdersToCreate = 0
+        var numOrdersToCancel = 0
```

Comment:
added logic to case for orders implicitly being cancelled by user's current form input